.github/workflows/prerelease.yml

```diff
@@ -5,36 +5,46 @@ on:
     branches:
       - main
 
+permissions:
+  contents: write
+
 env:
+  GPG_MAIL: 'infrastructure-eng@newrelic.com'
   OHAI_PFX_CERTIFICATE_BASE64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }} # base64 encoded
   OHAI_PFX_PASSPHRASE:  ${{ secrets.OHAI_PFX_PASSPHRASE }}
-  GO_VERSION: '1.18'
+  GORELEASER_VERSION: 'v1.13.1'
 
 ###
 ###   PRE-RELEASE CREATION # Creates a pre-release if needed and loads variables for next jobs
 ###
 name: Create Pre-release
 jobs:
-  create_prerelease:
-    name: Create Pre-release
+  exporter_variables:
+    name: Test exporter e2e and output exporter variables
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: ${{env.GO_VERSION}}
-      - name: Checkout code
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: nri-config-generator/go.mod
+
```

Comment:
[super nit] sometimes in this file we add extra lines and sometimes we do not 😅 (feel 100% free to ignore this)

Reply:
I tried to make it a bit more consistent: [caada7e](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/caada7e2baaa4d1b5859d04712091eb97fc86ccc)

Reply:
thanks